packages/front-end/components/Settings/ApiKeys.tsx

```diff
@@ -9,10 +9,15 @@ import ApiKeysModal from "./ApiKeysModal";
 import Link from "next/link";
 import { DocLink } from "../DocLink";
 
+type ApiKeys = {
+  [key: string]: string;
+};
```

Comment:
Use a more descriptive type name.  Based on this name, I would think it was an array of ApiKey objects.